integration/kubernetes/cleanup_bare_metal_env.sh

```diff
@@ -35,6 +35,9 @@ fi
 #cleanup stale file under /run
 sudo sh -c 'rm -rf /run/flannel'
 
+#cleanup the kubernetes apt list file
+sudo rm /etc/apt/sources.list.d/kubernetes.list
```

Comment:
@UnmeshDeodhar if `/etc/apt/sources.list.d/kubernetes.list` doesn't exist then this command will fail. Could you add the `-f` flag to ensure it won't fail?

Reply:
Fixed it!